WalletWasabi.Gui/Tabs/WalletManager/LoadWalletView.xaml

```diff
@@ -64,8 +64,9 @@
               <controls:Spinner Height="15" Width="15" Margin="0 0 8 0" />
               <TextBlock Text="Looking for hardware wallets..." FontWeight="Bold" />
             </StackPanel>
-            <TextBlock Text="Make sure you set up and logged into the device, usually with a PIN. Some hardware wallets are picky about their USB cables, so you may want to try out multiple ones." TextWrapping="Wrap" />
-            <TextBlock IsVisible="{Binding IsLinux}" Text="Also make sure that you properly applied udev rules. These are necessary for the devices to be reachable on Linux environments:" TextWrapping="Wrap" />
+            <TextBlock Text="Be sure to set up and log into the device, usually with a PIN." TextWrapping="Wrap" />
```

Comment:
While I'm not a native English speaker, this change sounds strange to me.

Reply:
English is fine here